test_autor.py

```diff
@@ -3,6 +3,17 @@
 
 class TestAutor(TestCase):
 
+    def test_quando_ImprimeAutor_retorna_SemErro(self):
+        autor = Autor("Igor",'igor.nascimento@caelum.com.br', 400 * "I")
```

Comment:
Confesso que esse `400 * I` é bem experto. Acredito ser até desnecessário, se não for para causar o erro do limite de caracteres.

Reply:
Entendi, coloquei isso quando estava verificando se funcionaria com 400, mas não com 401.

Reply:
Entendi